Mono.CairoWarp/WarpSpiral.cs

```diff
@@ -13,9 +10,6 @@ public class WarpSpiral : Warp
 
 		public WarpSpiral(Rectangle rect)
 		{
-			if (rect == null)
-				throw new ArgumentNullException("rect");
-
```

Comment:
why?

Reply:
Rectangle is a struct, never will be null